crates/db_schema/src/source/post.rs

```diff
@@ -37,18 +37,18 @@ pub struct PostForm {
   pub creator_id: PersonId,
   pub community_id: CommunityId,
   pub nsfw: Option<bool>,
-  pub url: Option<DbUrl>,
-  pub body: Option<String>,
+  pub url: Option<Option<DbUrl>>,
+  pub body: Option<Option<String>>,
   pub removed: Option<bool>,
   pub locked: Option<bool>,
   pub published: Option<chrono::NaiveDateTime>,
   pub updated: Option<chrono::NaiveDateTime>,
   pub deleted: Option<bool>,
   pub stickied: Option<bool>,
-  pub embed_title: Option<String>,
-  pub embed_description: Option<String>,
-  pub embed_video_url: Option<DbUrl>,
-  pub thumbnail_url: Option<DbUrl>,
+  pub embed_title: Option<Option<String>>,
+  pub embed_description: Option<Option<String>>,
+  pub embed_video_url: Option<Option<DbUrl>>,
+  pub thumbnail_url: Option<Option<DbUrl>>,
   pub ap_id: Option<DbUrl>,
   pub local: Option<bool>,
```

Comment:
I cant tell why some of these are Option<Option<>>, and others are just Option<>.

Reply:
Has to do with DB fields.

- Nullable field w/o default -> Option<Option<>>
- Nullable field w/ default -> Option
- Required field w/ default -> Option

Reply:
Would be good if there was a ci check or something to check that this is followed in all cases. Particularly when fields get added in future prs.

Reply:
Unfortunately the diesel schema doesn't keep track of default SQL values, only Nullables, so there'd be no way to check that. When doing this I have to bring up a SQL window to check them manually. Its just something I have to keep in mind when adding columns.